rust-runtime/smithy-types/src/instant/mod.rs

```diff
@@ -97,6 +97,7 @@ impl Instant {
         self.to_chrono_internal()
     }
 
+    #[cfg(feature = "chrono-conversions")]
     fn to_chrono_internal(&self) -> DateTime<Utc> {
```

Comment:
this private function can't be behind feature gate because we need this function defined in all cases for the library to compile

Reply:
Good catch! Fixed.